src/Searchable.php

```diff
@@ -60,7 +60,7 @@ public function queueMakeSearchable($models)
         }
 
         if (! config('scout.queue')) {
-            return $models->first()->searchableUsing()->update($models);
+            dispatch_now(new MakeSearchable($models));
```

Comment:
The return is to prevent it from being fired twice. With this change the command will be fired again with the dispatch below it.

Reply:
We can dispatch in 'else' section instead of immediately return
```
        if (! config('scout.queue')) {
            dispatch_now(new MakeSearchable($models));
        } else {
            dispatch((new MakeSearchable($models))
                    ->onQueue($models->first()->syncWithSearchUsingQueue())
                    ->onConnection($models->first()->syncWithSearchUsing()));
    }
```

Reply:
I've added new commit in the local repo but it didn't appear in the PR. Can I open a new one?

Reply:
yes